client/src/api/api.ts

```diff
@@ -131,3 +205,9 @@ export const topResultsResponse = async (category: string, subtype?: string) =>
 
   return results.data;
 };
+
+export const singleResultResponse = async (category: string, id: number) => {
```

Comment:
you could give this a return type (```Promise<Anime/SingleViewAnimeResult>```) also, but that depends if we're going to do separate routes for `/anime/:id` and `/manga/:id`